xray/services/utils/ignorerulebody.go

```diff
@@ -0,0 +1,65 @@
+package utils
+
+import "time"
+
+type IgnoreRuleParams struct {
+	Notes         string        `json:"notes"`
+	ExpiresAt     time.Time     `json:"expires_at,omitempty"`
+	IgnoreFilters IgnoreFilters `json:"ignore_filters"`
+}
+
+type IgnoreRuleBody struct {
+	Id            string        `json:"id,omitempty"`
+	Author        string        `json:"author,omitempty"`
+	Created       time.Time     `json:"created,omitempty"`
+	IsExpired     bool          `json:"is_expired,omitempty"`
+	Notes         string        `json:"notes"`
+	ExpiresAt     time.Time     `json:"expires_at,omitempty"`
+	IgnoreFilters IgnoreFilters `json:"ignore_filters"`
```

Comment:
```suggestion
	IgnoreRuleParams
```
Same as `IgnoreRuleParams`